src/tslint-cli.ts

```diff
@@ -22,8 +22,8 @@ const fs = require("fs");
 const optimist = require("optimist")
     .usage("usage: $0")
     .check((argv: any) => {
-        // at least one of file or help or version must be present
-        if (!(argv.f || argv.h || argv.v)) {
+        // at least one of file, help, version or unqualified argument must be present
+        if (!(argv.f || argv.h || argv.v || argv._.length > 0)) {
```

Comment:
let's remove `argv.f` here and change the error message to "Missing file(s) to lint"